### PR TITLE
Games page container

### PR DESCRIPTION
**Pull request by tmansuroglu**
#34 

This PR includes games page container and functionality for #75 .

### CODE REVIEW

**Review by ibrahimaljarad — APPROVED** [review]
i think it clear and easy to read

**Review by ssinikoglu — APPROVED** [review]

File: src/containers/GamesPage/index.jsx [hunk 1]
```diff
@@ -1,12 +1,41 @@
...
+import matchingGame from "../../images/gameScreenshots/matchingGame.svg";
 
 const GamesPage = () => {
+  const [t] = useTranslation();
+  // agreed naming is as follows:
+  // image file name: memoryGame
+  // game title key in locales: memoryGame.title
+  // game route: memorygame
+  const gameScreenshots = [memoryGame, puzzleGame, matchingGame, quizGame];
+  const reducer = (accObj, gameScreenshotDir) => {
```
Comment (louisrli) [inline src/containers/GamesPage/index.jsx]: just use a for loop

**Review by louisrli — APPROVED** [review]

**Review by louisrli — CHANGES_REQUESTED** [review]
for loop is better than reduce in readability in 99 percent of cases

**Comment by tmansuroglu** [comment]
@louisrli reduce replaced with for loop. PTAL

File: src/containers/GamesPage/index.jsx [hunk 2]
```diff
@@ -1,12 +1,40 @@
...
+import quizGame from "../../images/gameScreenshots/quizGame.svg";
+import matchingGame from "../../images/gameScreenshots/matchingGame.svg";
 
 const GamesPage = () => {
+  const [t] = useTranslation();
+  // agreed naming is as follows:
+  // image file name: memoryGame
+  // game title key in locales: memoryGame.title
+  // game route: memorygame
+  const gameScreenshots = [memoryGame, puzzleGame, matchingGame, quizGame];
```
Comment (louisrli) [inline src/containers/GamesPage/index.jsx]: GAME_SCREENSHOTS and line 9

File: src/containers/GamesPage/index.jsx [hunk 3]
```diff
@@ -1,12 +1,40 @@
...
+  const gamePostersObj = {};
+  // gameScreenshotDir example /static/media/memoryGame.84fe967a.svg
+  for (const gameScreenshotDir of gameScreenshots) {
+    const screenshotDirParts = gameScreenshotDir.split("/");
+    // example memoryGame.84fe967a.svg
+    const lastScreenshotDirPart =
+      screenshotDirParts[screenshotDirParts.length - 1];
+    // example memoryGame
```
Comment (louisrli) [inline src/containers/GamesPage/index.jsx]: put a colon after example, here and everywhere else

**Review by louisrli — CHANGES_REQUESTED** [review]

File: src/containers/GamesPage/index.jsx (hunk 3 above)
Comment (tmansuroglu) [reply src/containers/GamesPage/index.jsx]: PTAL

File: src/containers/GamesPage/index.jsx (hunk 2 above)
Comment (tmansuroglu) [reply src/containers/GamesPage/index.jsx]: GAME_SCREENSHOTS  implemented. PTAL. But what do you mean by line 9? It is empty

**Comment by louisrli** [comment]
I meant move it to line 9.

**Comment by tmansuroglu** [comment]
@louisrli  PTAL

**Review by louisrli — APPROVED** [review]

**Comment by tmansuroglu** [comment]
@Ammar-64 ready for merge

**Pull request merged**